php-frontend/src/test/java/org/sonar/php/tree/impl/expression/AnonymousClassTreeTest.java

```diff
@@ -19,17 +19,30 @@
  */
 package org.sonar.php.tree.impl.expression;
 
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
 import org.sonar.php.PHPTreeModelTest;
+import org.sonar.php.tree.impl.PHPTree;
+import org.sonar.php.utils.Assertions;
 import org.sonar.plugins.php.api.tree.Tree.Kind;
 import org.sonar.plugins.php.api.tree.expression.AnonymousClassTree;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
-public class AnonymousClassTreeTest extends PHPTreeModelTest {
+class AnonymousClassTreeTest extends PHPTreeModelTest {
+  @Test
```

Comment:
Do we want to have a space between the declaration and the first annotation?

Reply:
You mean in the code style? I would say not, but I may be failing to see a good reason

Reply:
I don't have a strong opinion on this. Please ignore my comment.